jetkit/api/auth/__init__.py

```diff
@@ -66,6 +69,24 @@ def login(email: str, password: str):
             abort(401, message="Wrong user name or password")
         return auth_response_for_user(user)
 
+    @blp.route("sign-up", methods=["POST"])
+    @blp.response(user_schema)
+    @blp.arguments(LoginRequest, as_kwargs=True)
+    def sign_up(email: str, password: str):
+        """Sign up with email and password. Possibly add other fields later."""
+        cleaned_email = email.strip().lower()
```

Comment:
Email usernames (unlike email domain names) are case sensitive. If we do `.lower()` here, it won't be possible to register users with emails `foobar@gmail.com` and `fooBar@gmail.com`, which are totally different

Reply:
Roman, can you cite your source here? I've never heard of that. You're saying I could register Rkyslyy@gmail.com even if you have rkyslyy@gmail.com?
I'm almost certain it's correct to lowercase email addresses before storing them

Reply:
Huh, sorry my bad
Looking at other sources and trying to register Rkyslyy@gmail.com I see that it's the same thing

Please disregard the comment above